hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestDeleteWithInAdequateDN.java

```diff
@@ -296,14 +287,11 @@ public void testDeleteKeyWithSlowFollower() throws Exception {
         }
         return
             dnStateMachine.getCommandDispatcher()
-                .getDeleteBlocksCommandHandler().getInvocationCount() >= 1;
+                .getDeleteBlocksCommandHandler().getInvocationCount() >= 0;
```

Comment:
This will always be true (>=0),  GenericTestUtils.waitFor(() {..} is not required. Command will not be invoked as one DN is down.

Reply:
Done